src/command_spec.rs

```diff
@@ -93,6 +117,7 @@ mod tests {
                         String::from("--option"),
                         String::from("value"),
                     ],
+                    envs: vec![],
```

Comment:
Could we also test for environment variables here? Not by using `format!` but instead by using `Command::{get_program, get_args, get_envs}`.